apps/server/src/modules/room/api/room.uc.ts

```diff
@@ -236,6 +244,38 @@ export class RoomUc {
 		return roleName;
 	}
 
+	public async addExternalPersonByEmailToRoom(
```

Comment:
I dislike having this as its own endpoint seperate from the registration invitation, but I guess that is a requirement we currently cant do anything about...